### PR TITLE
Public Pages - Improve routing and app selection

### PR DESCRIPTION
**Pull request by adamaziz15**
@ouranos As suggested, I added a conditional for routing based on tenant config

### CODE REVIEW

**Comment by adamaziz15** [comment]
@ouranos Also updated tenant config so that app list is updated to full list after initialization

**Comment by ouranos** [comment]
See #417 for the original PR

File: core/lib/generators/mno_enterprise/install/install_generator.rb [hunk 1]
```diff
@@ -117,7 +117,11 @@ def notify_about_routes
     mount MnoEnterprise::Engine, at: '/mnoe', as: :mno_enterprise
   end
 
-  root to: redirect('dashboard/')
+  if Settings.dashboard.public_pages.enabled
+    root to: redirect('dashboard/')
+  else
+    root to: redirect('mnoe/auth/users/sign_in')
+  end
```
Comment (ouranos) [inline core/lib/generators/mno_enterprise/install/install_generator.rb]: This is weird, it's already done in #425
That's the exact same change, I'm surprised that Github is showing it

**Comment by ouranos** [comment]
Incorporated in #458

**Pull request closed without merging**